coordinape/coordinape
# Implement createVault Hasura Action

test plan:
1. `docker compose --profile dev down -v` && `yarn docker:start`
2. vercel dev
3. browse to http://localhost:3000/admin/vaults
4. create a new vault with any token and observe it populate

Here's a template mutation for ensuring the endpoint is secure:
```gql
mutation testCreateVault {
  createVault(payload: {chain_id: 1338, org_id: 2, vault_address: "0x<address>"}) {
    id
    vault {
      id
      created_at
      created_by
      decimals
      org_id
      simple_token_address
      symbol
      token_address
      updated_at
      vault_address
    }
  }
}

```

try different org IDs that the user might not have access to, and test out an invalid vault address.
